test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -1141,6 +1149,15 @@ protected void refreshAllIndices() throws IOException {
         client().performRequest(refreshRequest);
     }
 
+    protected static RefreshResponse refresh(String index) throws IOException {
```

Comment:
Reusing server module's CreateIndexResponse, AcknowledgedResponse for RefreshResponse tests. 
It isn't part of the hlrc code base, but does provide us in tests the capabilities to easily parse the responses for the related apis.